assets/css/hero.scss

```diff
@@ -0,0 +1,21 @@
+.usa-hero {
```

Comment:
wondering about adding component based theme css for things like hero, header, footer, etc... 
It might make things a bit more digestible and modular.

Reply:
Yeah I think thats a good idea - it'll make it clear that it's separate from the main USWDS library. Typically the way I see this normally being structured is having each component in a partial like `_hero.scss` and then rolling up all of the components into a main entry file like `custom.scss`, which is in turn imported into the main.css file. What do you think of that approach?